### DISCUSSION TITLE
What's the rate limit?

### DISCUSSION THREAD

**Discussion by AleksaDjordjic** [simple-login/app#623] [Self-hosting question]
We're utilizing the API to quickly make a lot of aliases, problem is, we're also quickly triggering the rate limit.
What's the current rate limit and can it be changed? If not, any plans to make it an option that can be turned on/off?

**Comment by nguyenkims** [comment]
The rate limit is there to avoid abuses and its setting can be changed in the SaaS version anytime. Are you using the API on a self-hosted instance?

**Reply by AleksaDjordjic** [reply]
We're using a self-hosted instance, just followed the readme to get it up and running with Docker
Its a private instance, so we're not really concerned much about people abusing it - Its us who are using it, and we'd like to, well, not be limited

**Reply by nguyenkims** [reply]
Oh I see. You can also import the aliases from a .csv file which doesn't have any rate limiting. The rate limit is currently "hardcoded" and can't be turned off.

**Reply by wonderwareguru** [reply]
Greetings, you know it's really underhanded to advertise a business service as unlimited, only to be hit with a limitation on the backend. Honestly, you're selling a service for unlimited email aliasing, did you not expect people to use your service for bulk sign-up work? I should be able to create and dispose of an "unlimited" amount of aliases without restriction. I didn't see any disclosures next to "Unlimited aliases" for your $30 plan, otherwise I would have taken my business elsewhere. I was willing to pay more such an "unlimited" service. I guess when it's too good to be true, it is. Can I cancel my year subscription and get my money back, pro rated? 1 day usage.
